ReactiveFeedback/Feedback.swift

```diff
@@ -4,7 +4,21 @@ import enum Result.NoError
 
 public struct Feedback<State, Event> {
     public let events: (Scheduler, Signal<State, NoError>) -> Signal<Event, NoError>
+    
+    /// Creates arbitrary a Feedback, by transforming sequence of State to sequence of Events that mutate the State
```

Comment:
Creates an* arbitary Feedback, by transforming a* sequence of `State` to a* sequece of `Events` that mutate the `State`